tests/e2e/test_rosacli_cluster_post.go

```diff
@@ -371,6 +367,97 @@ var _ = Describe("Healthy check",
 				Expect(err).To(BeNil())
 				Expect(jsonData.DigString("nodes", "compute_machine_type", "id")).To(Equal(clusterConfig.Nodes.ComputeInstanceType))
 			})
+
+		It("with policy path will work - [id:72525]", labels.Runtime.Day1Post, labels.High,
+			func() {
+				path := profile.AccountRoleConfig.Path
+				if path == "" {
+					Skip("No account-role path defined. Skipping ...")
+				}
+
+				By("Retrieve cluster description")
+				clusterDesc, err := clusterService.DescribeClusterAndReflect(clusterID)
+				Expect(err).To(BeNil())
+
+				By("Check account roles path")
+				Expect(clusterDesc.STSRoleArn).To(ContainSubstring(path))
+				Expect(clusterDesc.SupportRoleARN).To(ContainSubstring(path))
+				for _, roles := range clusterDesc.InstanceIAMRoles {
+					for _, role := range roles {
+						Expect(role).To(ContainSubstring(path))
+					}
+				}
+
+				By("Check operator roles path")
+				for _, role := range clusterDesc.OperatorIAMRoles {
+					Expect(role).To(ContainSubstring(path))
+				}
+			})
+
+		It("with multiAZ will work - [id:75535]", labels.Runtime.Day1Post, labels.High,
+			func() {
+				By("Retrieve cluster description")
+				clusterDesc, err := clusterService.DescribeClusterAndReflect(clusterID)
+				Expect(err).To(BeNil())
+
+				By("Check Data plane")
+				if isHosted {
+					for _, plane := range clusterDesc.Availability {
+						for planeKey, planeValue := range plane {
+							if planeKey == "Data Plane" {
+								if profile.ClusterConfig.MultiAZ {
+									Expect(planeValue).To(Equal("MultiAZ"))
+								} else {
+									Expect(planeValue).To(Equal("SingleAZ"))
+								}
+							}
+						}
+					}
+				} else {
+					Expect(clusterDesc.MultiAZ).To(Equal(profile.ClusterConfig.MultiAZ))
+				}
+
+				By("Check compute machine type")
+				jsonData, err := clusterService.GetJSONClusterDescription(clusterID)
+				Expect(err).To(BeNil())
+				Expect(jsonData.DigString("nodes", "compute_machine_type", "id")).To(Equal(clusterConfig.Nodes.ComputeInstanceType))
```

Comment:
Could you please also chek that 3 default machinepools are created with 1 replicas to each on given subnet ?

Reply:
gonna add this